takwimu/views.py

```diff
@@ -275,10 +275,20 @@ def get_context_data(self, *args, **kwargs):
         data = json.load(json_data)
 
         ## get profileData aka summaries from wagtail
-        summary_data = ProfileData.objects.filter(country_iso_code=self.geo.geo_code).values('id', 'chart_id', 'summary')
+        summary_data = ProfileData.objects.filter(
+            country_iso_code=self.geo.geo_code).values('id', 'chart_id',
+                                                       'summary')
 
         page_context = {"profile_data_summaries": list(summary_data)}
 
+        # get data by topic page text description
+        try:
+            page = DataByTopicPage.objects.get(country=self.geo)
+            page_context['data_by_topic_description'] = page.description
+        except DataByTopicPage.DoesNotExist:
+            page_context[
+                'data_by_topic_description'] = f"What lorem ipsum dolor sit amec cat this is an introduction to the demographics data topic for {self.geo.name} and should be editorially curated & populated via the backend."
+
```

Comment:
I think it's best to forbid `blank` rather than handling exception here like this.